docs/design/actions/move.md

```diff
@@ -0,0 +1,29 @@
+# movePod #
+[This project](https://github.com/songbinliu/movePod) demonstrates a method that can move pods, which either are created by ReplicationController, or by ReplicaSet(which may be created by Deployment).
+
+# Method #
+**1.** set the schedulerName of the parent object (ReplicationController, or ReplicaSet) of the pod to a **invalid scheduler**; 
+
+**2.** move the pod by [**Copy-Delete-Create**](https://github.com/songbinliu/movePod/blob/master/util.go#L273) steps, and uses the **Binding-on-Creation** way by assigning [pod.Spec.NodeName](https://github.com/kubernetes/client-go/blob/master/pkg/api/v1/types.go#L2470)
```

Comment:
"Create" seems better than "Restart".   Because we indeed create a new Pod with the old PodSpec.